contrib/core/tests/test_action_sendmail.py

```diff
@@ -38,6 +39,10 @@
 MOCK_EXECUTION.id = "598dbf0c0640fd54bffc688b"
```

Comment:
Not relevant to your changes, but do we need this hardcoded id here?

Reply:
I don't see a problem with creating a mock using hard-coded values.